### ISSUE TITLE
GPU load

### ISSUE THREAD

**Issue by alexcyn** [oyiptong/compute-pressure#9]
It would be nice to understand GPU load as well, as in some cases this is a bottleneck, not a CPU.
For example, when on low notebook in a video conference I screenshare external 4k screen, built-in internal GPU is 50-60% busy, CPU is 30% busy, and everything starts working very slowly - even mouse becomes jerky.

**Comment by kenchris** [comment]
Yes, we could support this in the API using someting like entryTypes (https://www.w3.org/TR/performance-timeline/#dom-performanceobserverinit) ie "cpu", "gpu".

Thought level 1 is CPU only, we should make sure it can be extended to support other sources in the future

**Comment by anssiko** [comment]
In video conferencing use cases, the bottleneck could also be the camera capture device. For example, iOS/iPadOS provide `AVCaptureDevice.SystemPressureState` API: https://developer.apple.com/videos/play/wwdc2021/10047/?time=1698

It is perhaps a good idea to document possible future extensions somewhere in the specification.
